examples/FNCS/federate1.yml

```diff
@@ -8,13 +8,17 @@ time_delta: 1s
 sim_length: 60s
 # Optional stop_agent_when_sim_complete default False
 stop_agent_when_sim_complete: True
+# Optional stop_agent_when_sim_complete default False
```

Comment:
I think this comment is not in the right place...subscription_topic is below this..

Reply:
yes, I create a new pull request to resolve it